spec/facter/resolvers/processors_spec.rb

```diff
@@ -45,6 +48,12 @@
 
         expect(result).to eq(speed)
       end
+
+      it 'returns extensions supported' do
+        result = Facter::Resolvers::Linux::Processors.resolve(:extensions)
```

Comment:
Tests are failing with:

```
  1) Facter::Resolvers::Linux::Processors when on x86 architecture when cpuinfo file is readable returns extensions supported
     Failure/Error: expect(result).to eq(extensions)

       expected: ["x86_64", "x86_64-v1", "x86_64-v2", "x86_64-v3"]
            got: ["i386"]
```

You'll want to stub the call to `Facter::Resolvers::Uname.resolve(:processor)` so it returns the expected values.

Reply:
I feel like the current crop of test failures isn't me...